run_service.sh

```diff
@@ -277,6 +277,32 @@ prompt_use_staking() {
     done
 }
 
+# Function to set or add a variable in the .env file and export it
+dotenv_set_key() {
```

Comment:
The parameter names have choosen to be similar to Python's `dotenv.set_key` method.